mlperf-data/original/generate_dataset_summaries.py

```
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)